app/interactors/save_person.rb

```diff
@@ -0,0 +1,28 @@
+class SavePerson < BaseInteractor
+  integer :id, default: nil
+  string :preferred_contact_method  # todo: string?
```

Comment:
[This example](https://github.com/maebeale/mutual-aid/pull/106/files/271fdcfc88b06ca0909620dbb9692746c532043c#diff-16f1073379df86cff8f30397d3d38296R72) covers that case, though not explicitly or directly. I could make it clearer by setting the `preferred_contact_method` right in the example instead of relying on it from the earlier defined `params`.

Reply:
i think it's ok. i'm still not used to reading tests, so pretty sure i just missed that. thanks!